test/Process/test_somd.py

```diff
@@ -37,6 +37,23 @@ def test_production(system):
     # Run the process and check that it finishes without error.
     assert run_process(system, protocol)
 
+def test_free_energy():
```

Comment:
Is this related to the disulfide stuff or is this just some additional unrelated test?

Reply:
Sorry, this was just an additional new test to make sure that `somd-freenrg` works, since it was broken by the Sire 2023 merge. I happened to be working on this branch at the time so added it here. It is unrelated to any of the new parameterisation functionality.